src/Form/index.tsx

```diff
@@ -165,7 +165,11 @@ export * from './grid/StyledContainer';
 export type { StyledContainerProps } from './grid/StyledContainer';
 
 export interface Props {
-  formName: string;
+  formId?: string;
+  /**
+   * @deprecated use formId instead
+   */
+  formName: string; // TODO: remove support for formName
```

Comment:
leaving `formName` required for now. I think it makes more sense to remove `formName` and make `formId` required when we're ready to release a version that makes `formId` the default. Also, makes sense to do that in a diff version since its a breaking change

Reply:
IMO it'd be better to make the formId the default now, it won't break anything but will be annoying and push users to switch

Reply:
cool, should we make `formName` optional then? so new users that are on the latest version don't write anything based on `formName` and existing users can remove the prop when they eventually refactor to `formId`